### PR TITLE
XRP currency support

### PR DESCRIPTION
**Pull request by lucas-maicoin**

### CODE REVIEW

**Comment by lihengl** [comment]
@lucas-maicoin 😃

**Comment by lucas-maicoin** [comment]
@lihengl Can we merge litecoin together ?

**Comment by lihengl** [comment]
I was about to push an identical commit...

File: config/currency_non_iso.json [hunk 1]
```diff
@@ -27,6 +27,34 @@
     "thousands_separator": ",",
     "iso_numeric": ""
   },
+  "ltc": {
+    "priority": 100,
+    "iso_code": "LTC",
+    "name": "Litecoin",
+    "symbol": "Ł",
+    "alternate_symbols": [ ],
+    "subunit": "N/A",
```
Comment (lihengl) [inline config/currency_non_iso.json]: "subunit": "Litoshi"

Comment (lucas-maicoin) [reply config/currency_non_iso.json]: updated

**Pull request merged**